shared/js/dataDriver.js

```diff
@@ -149,6 +152,12 @@ loop.DataDriver = function() {
             console.log(`Error handling EventSource.put: ${event.data}`, ex);
           }
           break;
+        case "error":
+          setTimeout(
+            () => this._connectToRoom(this._roomToken),
+            this._retryTimeout);
+          this._retryTimeout *= 2;
```

Comment:
My earlier comment still is unanswered about testing the backoff. How are you getting multiple `error` events?